main.go

```diff
@@ -76,7 +76,12 @@ func tcpEventCb(event tcpEvent) {
 }
 
 func main() {
-	b := bpf.NewBpfPerfEvent("kernel/trace_output_kern.o")
+	if len(os.Args) != 2 {
+		fmt.Fprintf(os.Stderr, "Usage: %s /ebpf/${DISTRO}/x86_64/$(uname -r)/ebpf.o\n", os.Args[0])
```

Comment:
you mean `${GOPATH}/src/github.com/kinvolk/tcptracer-bpf/ebpf/${DISTRO}/x86_64/$(uname -r)/ebpf.o`